### ISSUE TITLE
ETAs not removed by remove_iov

### ISSUE THREAD

**Issue by joaoabr** [pharmpy/pharmpy#411]
Tested by manually importing moxonidine model. OMEGAs are removed, but not etas which leads to biased results.

**Comment by pharmpy-dev-123** [comment]
@joaoabr @rikardn Was this an issue about ETAs set to 0 but not being removed or was it something else?

**Comment by joaoabr** [comment]
This seems to have been fixed already (pharmpy 0.72.2), as the same example is working now (see below). I believe that previously the ETA()s were not removed from $PK. 


model %>% 
    print_model_code()

```
$PROBLEM    MOXONIDINE PK ANALYSIS
$ABBREVIATED DERIV2=NO COMRES=6
$INPUT      ID VISI XAT2=DROP DGRP=DROP DOSE=DROP FLAG=DROP ONO=DROP
            XIME=DROP DVO=DROP NEUY=DROP SCR=DROP AGE SEX NYHA=DROP WT
            COMP ACE DIG DIU NUMB=DROP TAD TIME VIDD=DROP CLCR AMT SS
            II VID CMT=DROP CONO=DROP DV EVID=DROP OVID=DROP
$DATA       mx19.dat IGNORE=#
$SUBROUTINE ADVAN2 TRANS1
$PK

;-----------COVARIATES----------
   VIS3               = 0
   IF(VISI.EQ.3) VIS3 = 1
   VIS8               = 0
   IF(VISI.EQ.8) VIS8 = 1
;----------IIV-------------------
   ETC     = EXP(ETA(1))
   ETCL    = ETA(1)
   ETEV    = ETA(2)
   ETV     = EXP(ETA(2))
   EKA     = EXP(ETA(3))
   ETKA    = ETA(3)
;----------IOV--------------------
   
   KPCL  = EXP(VIS3*ETA(4)+VIS8*ETA(5))
   KPKA  = EXP(VIS3*ETA(6)+VIS8*ETA(7))
   KPLAG = VIS3*ETA(8)+VIS8*ETA(9)
   KAPC  = VIS3*ETA(4)+VIS8*ETA(5)

   TVCL  = THETA(1)*(1+THETA(6)*(CLCR-65))
   TVV   = THETA(2)*WT

   CL    = TVCL*ETC *KPCL
   V     = TVV*ETV  
   KA    = THETA(3)*EKA *KPKA
   LAG   = THETA(4)
   PHI   = LOG(LAG/(1-LAG))
   ALAG1 = EXP(PHI+KPLAG)/(1+EXP(PHI+KPLAG))
   K     = CL/V
   S2    = V

$ERROR


     IPRED = LOG(.025)
     W     = THETA(5)
     IF(F.GT.0) IPRED = LOG(F)
     IRES  = IPRED-DV
     IWRES = IRES/W
     Y     = IPRED+ERR(1)*W

$THETA  (0,27.5)       ;            TVCL
$THETA  (0,1.565)       ;             TVV
$THETA  (0,2.1)       ;            TVKA
$THETA  (0,.254)       ;             LAG
$THETA  (0,.23)       ;         RES ERR
$THETA  (0,.008,.02941)       ;      CRCL on CL
$OMEGA  BLOCK(2) .3
 .1
 .3  ; IIV (CL-V)
$OMEGA  BLOCK(1) .3  ;     IIV KA
$OMEGA  BLOCK(1) .3  ; IOV CL VISI=3
$OMEGA  BLOCK(1) SAME

$OMEGA  BLOCK(1) .3  ; IOV KA VISI=3
$OMEGA  BLOCK(1) SAME

$OMEGA  BLOCK(1) 0 FIX  ; IOV LAG VISI=3
$OMEGA  BLOCK(1) SAME

$SIGMA  1  FIX
$ESTIMATION METHOD=1 INTER MAXEVALS=9990 PRINT=2 POSTHOC
;$COVARIANCE PRINT=E
;$TAB ID TAD IPRED IWRES            ONEHEADER NOPRINT FILE = sdtab111
;$TAB VID TVCL TVV V CL KA ALAG1 ETCL ETEV KAPC KPLAG 
;     ONEHEADER NOPRINT FILE = patab111
;$TAB VID AGE WT CLCR               ONEHEADER NOPRINT FILE = cotab111
;$TAB VID SEX ACE DIG DIU COMP VISI ONEHEADER NOPRINT FILE = catab111
```


#remove IOV
remove_iov(model)
model %>% 
    print_model_code()

```
$PROBLEM    MOXONIDINE PK ANALYSIS
$ABBREVIATED DERIV2=NO COMRES=6
$INPUT      ID VISI XAT2=DROP DGRP=DROP DOSE=DROP FLAG=DROP ONO=DROP
            XIME=DROP DVO=DROP NEUY=DROP SCR=DROP AGE SEX NYHA=DROP WT
            COMP ACE DIG DIU NUMB=DROP TAD TIME VIDD=DROP CLCR AMT SS
            II VID CMT=DROP CONO=DROP DV EVID=DROP OVID=DROP
$DATA       mx19.dat IGNORE=#
$SUBROUTINE ADVAN2 TRANS1
$PK

;-----------COVARIATES----------
   VIS3               = 0
   IF(VISI.EQ.3) VIS3 = 1
   VIS8               = 0
   IF(VISI.EQ.8) VIS8 = 1
;----------IIV-------------------
   ETC     = EXP(ETA(1))
   ETCL    = ETA(1)
   ETEV    = ETA(2)
   ETV     = EXP(ETA(2))
   EKA     = EXP(ETA(3))
   ETKA    = ETA(3)
;----------IOV--------------------
   

KPCL = 1
KPKA = 1
KPLAG = 0
KAPC = 0
   TVCL  = THETA(1)*(1+THETA(6)*(CLCR-65))
   TVV   = THETA(2)*WT

   CL    = TVCL*ETC *KPCL
   V     = TVV*ETV  
   KA    = THETA(3)*EKA *KPKA
   LAG   = THETA(4)
   PHI   = LOG(LAG/(1-LAG))
   ALAG1 = EXP(PHI+KPLAG)/(1+EXP(PHI+KPLAG))
   K     = CL/V
   S2    = V

$ERROR


     IPRED = LOG(.025)
     W     = THETA(5)
     IF(F.GT.0) IPRED = LOG(F)
     IRES  = IPRED-DV
     IWRES = IRES/W
     Y     = IPRED+ERR(1)*W

$THETA  (0,27.5)       ;            TVCL
$THETA  (0,1.565)       ;             TVV
$THETA  (0,2.1)       ;            TVKA
$THETA  (0,.254)       ;             LAG
$THETA  (0,.23)       ;         RES ERR
$THETA  (0,.008,.02941)       ;      CRCL on CL
$OMEGA  BLOCK(2) .3
 .1
 .3  ; IIV (CL-V)
$OMEGA  BLOCK(1) .3  ;     IIV KA
$SIGMA  1  FIX
$ESTIMATION METHOD=1 INTER MAXEVALS=9990 PRINT=2 POSTHOC
;$COVARIANCE PRINT=E
;$TAB ID TAD IPRED IWRES            ONEHEADER NOPRINT FILE = sdtab111
;$TAB VID TVCL TVV V CL KA ALAG1 ETCL ETEV KAPC KPLAG 
;     ONEHEADER NOPRINT FILE = patab111
;$TAB VID AGE WT CLCR               ONEHEADER NOPRINT FILE = cotab111
;$TAB VID SEX ACE DIG DIU COMP VISI ONEHEADER NOPRINT FILE = catab111
```

**Comment by pharmpy-dev-123** [comment]
@rikardn Can we close or was there something else?
